spirv_hlsl.hpp

```diff
@@ -23,6 +23,13 @@
 
 namespace spirv_cross
 {
+struct HLSLVertexAttr
+{
+	std::string name;
```

Comment:
This is a bit problematic, as this maps a *name* to a new semantic. However, you cannot rely on a name existing in the SPIR-V, e.g. if it has been stripped. Name information is purely debug info, and this interface should map location to a custom semantic rather than TEXCOORD#{location}. If your interfaces deal with names, it's trivial to map names to location.